libs/jsonforms-components/src/lib/Controls/AddressLookup/AddressLookUpControl.tsx

```diff
@@ -112,40 +119,122 @@ export const AddressLookUpControl = (props: AddressLookUpProps): JSX.Element =>
     setSuggestions([]);
     setErrors({});
   };
+
+  /* istanbul ignore next */
   const handleRequiredFieldBlur = (name: string) => {
-    if ((!data?.[name] || data?.[name] === '') && requiredFields?.includes(name)) {
-      const err = { ...errors };
-      err[name] = name === 'municipality' ? `city is required` : `${name} is required`;
+    const err = { ...errors };
+    if(data?.["city"] === undefined || data?.["city"] === ""){
+      err[name] = name === 'municipality' ? 'city is required' : ""
       setErrors(err);
-    } else {
+    }
+
+    if(!data?.[name] || data[name] === '' || data?.[name] === undefined){
+      err[name] = name === 'municipality' ? 'city is required' : `${name} is required`;
+      setErrors(err);
+    }
+
+    if(!data?.[name]){
```

Comment:
I think addressLine1 error is handle automatically, remove this part not affect test.